### PR TITLE
reset last_avatar_update column when avatars cache is cleared

### PR DESCRIPTION
**Pull request by prettyirrelevant**
## Description
- Whenever the cache of an avatar is cleared, `last_avatar_update` for the particular ENS name is set to `Timestamp(0)`.
- If the entire cache is cleared, `last_avatar_update` is set to `Timestamp(0)` for all ENS names present in the DB.

- `last_avatar_update` is now non-nullable and a default of 0 is set.
- A follow up from a previous PR is also included.

### CODE REVIEW

File: rotkehlchen/api/rest.py [hunk 1]
```diff
@@ -4253,11 +4253,20 @@ def clear_avatars_cache(self, avatars: Optional[list[str]]) -> Response:
                 if entry.is_file():
                     entry.unlink()
 
+            with self.rotkehlchen.data.db.user_write() as delete_cursor:
+                delete_cursor.execute('UPDATE ens_mappings SET last_avatar_update=?', (Timestamp(0),))  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/api/rest.py:4257]: Isn't it better to set it to null? What was the thinking of having it as null before @nebolax ?

File: rotkehlchen/tests/api/test_misc.py [hunk 2]
```diff
@@ -418,6 +420,20 @@ def test_cache_deletion(rotkehlchen_api_server):
     avatars_dir = icons_dir / 'avatars'
     avatars_dir.mkdir(exist_ok=True)
 
+    dbens = DBEns(rotkehlchen_api_server.rest_api.rotkehlchen.data.db)
+    with dbens.db.user_write() as write_cursor:
+        dbens.add_ens_mapping(
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_misc.py:425]: why is this needed in the test?

Comment (prettyirrelevant) [reply rotkehlchen/tests/api/test_misc.py:425]: Querying an avatar that's not present in the ens_mappings will raise an error.

File: rotkehlchen/api/rest.py (hunk 1 above)
Comment (nebolax) [reply rotkehlchen/api/rest.py:4257]: I used null to be more explicit

Comment (prettyirrelevant) [reply rotkehlchen/api/rest.py:4257]: Alright then. Will set it to NULL

File: rotkehlchen/tests/api/test_misc.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/tests/api/test_misc.py:425]: then you should have added a comment explaining this is why you are doing it.

**Review by LefterisJP — APPROVED** [review]
LGTM but not sure why the 2 tests failed.

**Comment by LefterisJP** [comment]
merging as discussed the tests are fixed elsewhere
